src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -99,6 +99,27 @@ paths:
         '404':
           $ref: '#/components/responses/404Error'
 
+  '/profile/{prisonNumber}/education-support-plan/review':
+    parameters:
+      - $ref: "#/components/parameters/prisonNumberPathParameter"
+    post:
+      summary: Review an Education Support Plan
+      description: |
+        Creates an Education Support Plan Review and updates the plan for the specified prisoner.
+
+        **Role Requirements:**
+        Access to this endpoint requires one of the following roles:
+          - `ROLE_SUPPORT_ADDITIONAL_NEEDS__ELSP__RW` (Read-Write)
+      tags:
+        - Support Plan
+      operationId: create-education-support-plan-review
+      responses:
+        '201':
+          description: Successfully created review (no response body)
+        '400':
```

Comment:
Minor, and perhaps this can be added when the impl is being done because I dont know the answer at the moment 🤣 , but I'm imagining there is some kind of error when the request body itself is all nice and valid 👍  (so not a 400), but the prisoner referenced in the URL does not have an ELSP to start with 🤔 
I dont know whether that would be a 404, or a 409, or perhaps something else? (I find it best to refer to [HTTP Status Codes in cat form](https://http.cat/) in these cases 🤣 )

I wonder also if there are cases where the prisoner has a plan, but a review is not necessary/required/supported? EG. they have a plan but are not in education? or something else? 🤔

Reply:
I'll make a note of this and tackle it when I know all the failure options.